src/common/io/bufferWrite.c

```diff
@@ -63,6 +44,25 @@ ioBufferWriteNew(Buffer *buffer)
     FUNCTION_DEBUG_RESULT(IO_BUFFER_WRITE, this);
 }
 
+/***********************************************************************************************************************************
+Write to the buffer
+***********************************************************************************************************************************/
+void
+ioBufferWrite(IoBufferWrite *this, Buffer *buffer)
```

Comment:
I don't see ioBufferWrite() ever being called. At a minimum there should be a unit test for it, no?

Reply:
It is unit tested via the IoWrite interface so you won't see any direct calls to it.  Since this is the way it will be used in practice it's the best way to test it.  And the IoWrite code is tested at the same time.